lib/membrane/compositor_element.ex

```diff
@@ -57,6 +76,9 @@ defmodule Membrane.VideoCompositor.CompositorElement do
 
     state = %{
       videos_positions: %{},
+      videos_z_values: %{},
+      videos_scales: %{},
+      pts_offsets: %{},
```

Comment:
I think you should gather all of these properties into the `VideoOptions` struct